tensorflow/contrib/learn/python/learn/datasets/synthetic_test.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+
+from tensorflow.contrib.learn.python.learn import datasets
+from tensorflow.contrib.learn.python.learn.datasets import synthetic
+
+class SyntheticTest(tf.test.TestCase):
+  """Test synthetic dataset generation
+  
+  TODO (tests):
```

Comment:
Yeah, I completely removed it - I forgot it was a list for myself for corner cases